README.md

```diff
@@ -30,6 +34,96 @@ Simple and efficient HTTP mock server with specification in `yaml`, `edn` or `Op
         }
 ```
 
+## With OpenAPI
```

Comment:
let's keep the documentation complete on the documentation page I started [here](https://avelino.run/projects/moclojer/)
source: https://github.com/avelino/avelino.run/blob/main/content/projects/moclojer.md

so we keep the readme clean with just a simple example of the _moclojer spec_

I know it is not the best place, but soon we will move to our own domain and no longer at [avelino.run](https://avelino.run)